data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/ComplexContingency.java

```diff
@@ -69,6 +69,8 @@ public void apply(Network network, ComputationManager computationManager) {
                 contingency.addElement(new HvdcLineContingency(contingencyElement.getId()));
             } else if (element instanceof BusbarSection) {
                 contingency.addElement(new BusbarSectionContingency(contingencyElement.getId()));
+            } else if (element instanceof DanglingLine) {
+                contingency.addElement(new DanglingLineContingency(contingencyElement.getId()));
```

Comment:
Did you merge the evol on Dangling lines inside this PR ? :smile:

Reply:
Well, perhaps we could keep it (if you want to I can remove it)

Reply:
Delete the changes I created a new small PR